### ISSUE TITLE
Migration issues using Multitenancy System

### ISSUE THREAD

**Issue by undjike** [JosephSilber/bouncer#641]
Hello Team,

First of all, thanks for the brillant job you achieved. It's really well implemented and time saving.

Nevertheless, I'm facing issues using the package in a multitenancy system where there is a permission management in each tenant database and also in the landlord database (means 02 levels of permission management implemented via 02 database connections).

First, I'm forced to use class name in the migration file generated by the package. When I convert it to anonymous class syntax, I get errors running migrations.

Second, I have a dedicated folder for tenant migrations and landlord migrations located the usual place. When running migrations (forced to use class name syntax), I get an error too.

![image](https://github.com/JosephSilber/bouncer/assets/57102409/933d43c9-1e24-44d8-90c7-5a752141fe2a)

So, my questions are:

1. Why not make use of anonymous class syntax for the package migration?
2. Is it possible to specify the connection used by the `Bouncer` facade?

**Comment by robinlecouteur** [comment]
I'm just cruising through, but have you tried naming the migration to TenantCreateBouncerTables or something? Also what is the exact error you get when using anonymous class?

**Comment by digitlimit** [comment]
@undjike have you resolved this issue?

**Comment by undjike** [comment]
I haven't. Do you face the same issue?

**Comment by digitlimit** [comment]
@undjike yes, building tenancy application with https://github.com/archtechx/tenancy

**Comment by undjike** [comment]
> @undjike yes, building tenancy application with https://github.com/archtechx/tenancy

I was using the same...

**Comment by digitlimit** [comment]
@undjike I am wondering if I can fix and submit a PR for this.
I believe anonymous migration classes work in Laravel 7 >=

**Comment by undjike** [comment]
I think it's from >=8.

**Comment by undjike** [comment]
https://laravel-news.com/laravel-anonymous-migrations
